app/controllers/instrument_relays_controller.rb

```diff
@@ -0,0 +1,38 @@
+class InstrumentRelaysController < ApplicationController
+
+  admin_tab :all
+  before_action :authenticate_user!
+  before_action :init_product
+  before_action :manage
+
+  layout "two_column"
+
+  # GET /facilities/:facility_id/instrument/:instrument_id/relays
+  def index
```

Comment:
This feels more like a `show` action to me, the relationship to relays is a `has_one`.  What do you think?

Reply:
yeah, I went back and forth on this. I think I ended up settling on index because you aren't selecting a specific relay to view but rather the tab is for all the relays an instrument has (which happens to be one). Happy to switch to show though, I don't feel super strongly.

Reply:
Thinking about this more, I feel like index also makes sense for the tab path because the instrument might not have a relay